node/resolvers/Routes/index.ts

```diff
@@ -530,13 +530,20 @@ export const Routes = {
       tradeName,
     })
 
+    const phoneNumberFormatted =
+      phoneNumber ||
+      clUser.phoneNumber ||
+      clUser.homePhone ||
+      `+1${'0'.repeat(10)}`
```

Comment:
Can you explain this change? It seems like we'll set the order form's `phone` to +10000000000 , which is a fake US number. Wouldn't it be better not to set any phone number if none exists?

Reply:
I still don't get it 100%, what makes the phone number required? Is it some specific logic for b2b suite's checkout?

And will this fake number show up in that user's profile after we set it?

Reply:
What I don't get is why phone number is required. From what I know, phone number is an optional field for VTEX checkout, I've even double checked the API being called here and confirmed that phone is optional. Same goes for `stateInscription`, by the way.

Is it different for B2B Suite somehow? I'm asking because it's important that my team understand these differences as we start taking care of it.